### PR TITLE
fix: disambiguate Quarkus launch config

### LINKED ISSUE #1045: Perception conflict between Red Hat and JetBrains Quarkus plugins
When using Intellij Idea ultimate, there are image clashes between Red Hat and JetBrains Quarkus plugins.
They have the same name and almost the same collor:

image (5)
image (4)
image (3)

But it gets worse!
If you use the light theme, there is no way to know who is who.

It would be nice to get a precise distiction between the two plugins.

### PR DESCRIPTION
**Pull request by fbricon**
Change the name of the Quarkus run configuration type to "Quarkus Dev Mode", to disambiguate from Intellij Ultimate's own config type.

Also fixed the dark mode icons, which unfortunately don't help with the confusion since they're identical to the ones used in Ultimate, but it's more correct now.

Screenshot 2023-09-05 at 15 24 22
Screenshot 2023-09-05 at 15 24 58

Partially fixes #1045

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ideaVersion = IC-2022.2
+ideaVersion = IU-2023.2.1
```
Comment (angelozerr) [inline gradle.properties:1]: dont update this file

File: src/main/java/com/redhat/devtools/intellij/quarkus/run/QuarkusRunConfiguration.java [hunk 2]
```diff
@@ -61,6 +61,7 @@
 
 public class QuarkusRunConfiguration extends ModuleBasedConfiguration<RunConfigurationModule, QuarkusRunConfigurationOptions> {
     private final static Logger LOGGER = LoggerFactory.getLogger(QuarkusRunConfiguration.class);
+    public static final String QUARKUS_CONFIGURATION = "Quarkus Configuration";
```
Comment (angelozerr) [inline src/main/java/com/redhat/devtools/intellij/quarkus/run/QuarkusRunConfiguration.java:64]: private

File: gradle.properties (hunk 1 above)
Comment (fbricon) [reply gradle.properties]: my bad

File: src/main/java/com/redhat/devtools/intellij/quarkus/run/QuarkusRunConfiguration.java (hunk 2 above)
Comment (fbricon) [reply src/main/java/com/redhat/devtools/intellij/quarkus/run/QuarkusRunConfiguration.java]: fixed

**Comment by angelozerr** [comment]
Thanks @fbricon !

**Comment by ia3andy** [comment]
Not sure why Quarkus Dev Mode?

**Comment by fbricon** [comment]
because it's run configurations to launch Quarkus in dev mode

**Comment by ia3andy** [comment]
ah ok 😅 I thought it was the plugin name ..

**Pull request merged**